File: Source/Tests/ForAlternetStudio/Resources/Scripter/CustomAssemblyTest.cs
using System;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;
using ExternalAssembly;

public class TestClass
{
    public void UseExternal()
    {
        CustomClass customClass = new CustomClass();
        customClass.TestMethod(1, true);
    }

    public static void Main()
    {
        TestClass F = new TestClass();
        F.UseExternal();
    }

    public TestClass()
    {
    }
}

File: Source/Tests/ForAlternetStudio/Resources/Scripter/CustomAssemblyTest.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports ExternalAssembly

Public Class TestClass
    Public Sub UseExternal()
        Dim customClass As CustomClass = New CustomClass()
        customClass.TestMethod(1, True)
    End Sub

    Public Shared Sub Main()
        Dim F As TestClass = New TestClass()
        F.UseExternal()
    End Sub

    Public Sub New()
    End Sub
End Class
